packages/babel-plugin-transform-es2015-destructuring/test/fixtures/destructuring/es7-object-rest/expected.js

```diff
@@ -3,7 +3,12 @@ var x = babelHelpers.objectWithoutProperties(z, []);
 var x = z.x;
 var y = babelHelpers.objectWithoutProperties(z, ["x"]);
 var x = z[x];
-var y = babelHelpers.objectWithoutProperties(z, [x]);
+var y = babelHelpers.objectWithoutProperties(z, [typeof x === "symbol" ? x : `${ x }`]);
```

Comment:
Wouldn't it be better to do this in the helper rather than bloating the code output?